components/FieldInput.test.jsx

```diff
@@ -15,11 +15,25 @@ it("renders empty inputs when no value is given", async () => {
   expect(selects.length).toBe(2);
 });
 
-it("updates the field type when changed", async () => {
+it("updates the field type when changed & is not valid", async () => {
   const mockHandleUpdate = jest.fn();
   render(<FieldInput value="" onUpdate={mockHandleUpdate} />);
 
   const fieldTypeSelect = screen.getAllByRole("combobox")[0];
   userEvent.selectOptions(fieldTypeSelect, "date");
-  expect(mockHandleUpdate).toHaveBeenCalledWith("date");
+  expect(mockHandleUpdate).toHaveBeenCalledWith(":date", false);
+});
+
+it("updates the field type when changed & is valid", async () => {
```

Comment:
@wasabigeek this test fails currently, I have tried different ways / hacks to get it working but couldn't :(

I have added the expectations that should be fulfilled.

Reply:
Ah I haven't figured out what's the best practice around this, but what is likely happening is:
- the `input`'s value is fixed, and `onUpdate` doesn't modify it
- typing "created_at" is essentially calling `onUpdate("c")`, `onUpdate("r")`, `onUpdate("e")`...  because the value doesn't update

Off the top of my head one could probably have 
```
let value = "";
const handleUpdate = (newValue) => { value = newValue }

...

expect(value).toEqual(...)
```
But it does feel like there must be a more idiomatic way to check that.

Reply:
Also, we're changing both the ~field~ attribute name and type in this test, not just the type.